po/it.po

```diff
@@ -260,9 +262,8 @@ msgid "Translation updates"
 msgstr "Aggiornamenti di traduzione"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:63
-#, fuzzy
 msgid "Updated Catalán and Spanish translations thanks to Mario Rodrigo"
-msgstr "Traduzioni Catalane aggiornate grazie a Mario Rodrigo."
+msgstr "Traduzioni Catalane aggiornate grazie a Mario Rodrigo"
```

Comment:
The Italian doesn’t match the English here.

Reply:
You're right! I didn't notice the difference with an another almost identical line. 😅